parm/atm/obs/config/seviri_m08.yaml

```diff
@@ -0,0 +1,328 @@
+- filter: Perform Action 
+  filter variables: 
+  - name: brightnessTemperature
+    channels: 6-7
+  where: 
+  - variable:
+      name: ClearSkyStdDev/brightnessTemperature_5
+    maxvalue: 0.5
+    minvalue: 0.4 
+    min_exclusive: true 
+  action:
+    name: inflate error
+    inflation factor: 1.14891
+
+- filter: Perform Action 
+  filter variables: 
+  - name: brightnessTemperature
+    channels: 6-7
+  where: 
+  - variable:
+      name: ClearSkyStdDev/brightnessTemperature_5
+    maxvalue: 0.6
+    max_exclusive: true 
+    minvalue: 0.5 
+    min_exclusive: true 
+  action:
+    name: inflate error
+    inflation factor: 1.29228
+
+- filter: Perform Action 
+  filter variables: 
+  - name: brightnessTemperature
+    channels: 6-7
+  where: 
+  - variable:
+      name: ClearSkyStdDev/brightnessTemperature_5
+    maxvalue: 0.7
+    max_exclusive: true 
+    minvalue: 0.6 
+  action:
+    name: inflate error
+    inflation factor: 1.49666  
+
+- filter: Perform Action 
+  filter variables: 
+  - name: brightnessTemperature
+    channels: 6-7
+  where: 
+  - variable:
+      name: ClearSkyStdDev/brightnessTemperature_5
+    minvalue: 0.7 
+  action:
+    name: inflate error
+    inflation factor: 1.51987
+
+# Step 11:
+- filter: Bounds Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 4-11
+  test variables:
+  - name: ObsFunction/NearSSTRetCheckIR
+    channels: 4-11
+    options:
+      channels: 4-11
+      use_flag: [ -1, 1, 1, -1,  -1, -1, -1, -1 ]
+  maxvalue: 1.0e-12
+  action:
+    name: reject
+
+# Step 12: Error inflation based on surface jacobian check
+- filter: Perform Action 
+  filter variables:
+  - name: brightnessTemperature
+    channels: 4-11
+  action:
+    name: inflate error
+    inflation variable:
+      name: ObsFunction/ObsErrorFactorSurfJacobianRad
+      channels: 4-11
+      options:
+        channels: 4-11
+        sensor: seviri_m08 
+        obserr_demisf: [0.01, 0.02, 0.03, 0.02, 0.03]
+        obserr_dtempf: [0.50, 2.00, 4.00, 2.00, 4.00]
+
+# Step 13: Cloud fraction cehck
+#          Reject channels 4, 6-11 if Cloud fraction (percent) > 2
+- filter: Domain Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 4, 6-11
+  where:
+  - variable:
+      name: MetaData/cloudAmount
+    maxvalue: 2
+
+# Step14: Final gross check
+- filter: Background Check
+  filter variables:
+  - name: brightnessTemperature
+    channels: 4-11
+  function absolute threshold:
+  - name: ObsFunction/ObsErrorBoundIR
+    channels: 4-11
+    options:
+      sensor: seviri_m08
+      channels: 4-11
+      obserr_bound_latitude:
+        name: ObsFunction/ObsErrorFactorLatRad
+        options:
+          latitude_parameters: [0.0, 0.0, 0.0, 0.0]
+      obserr_bound_transmittop:
+        name: ObsFunction/ObsErrorFactorTransmitTopRad
+        channels: 4-11
+        options:
+          channels: 4-11
+      obserr_boundax: [ 2.0, 4.0, 3.5, 2.0, 2.0, 2.0, 2.0, 3.0 ]
```

Comment:
@RussTreadon-NOAA here's your next issue, a typo. @azadeh-gh did you miss any commits when testing this that are not included in this PR?